modules/rds/main.tf

```diff
@@ -1,6 +1,6 @@
 module "rds" {
   source  = "terraform-aws-modules/rds-aurora/aws"
-  version = "9.0.0"
+  version = "6.2.0"
```

Comment:
why downgrade to 6 version? the latest version is `9.9.1`: https://registry.terraform.io/modules/terraform-aws-modules/rds-aurora/aws/latest

Reply:
It was failed when I tried but found the way to fix it by adding `create_db_subnet_group = true` to override the default `false`

Updated in [05e7c87](https://github.com/nvminhtue/tyson_devops_ic/pull/33/commits/05e7c877832233d991a9617ec18b92e10ddda455)
![image](https://github.com/user-attachments/assets/2fb5e143-1a17-49d9-99c8-bfc04fb77172)